Fails for me with the following error:

```
deno>    Compiling swc_ecma_transforms_react v0.25.0
deno> error[E0658]: or-patterns syntax is experimental
deno>     --> /build/deno-1.11.4-vendor.tar.gz/swc_ecma_transforms_optimization/src/simplify/expr/mod.rs:1347:34
deno>      |
deno> 1347 | ...                   Some(Expr::Lit(..) | Expr::Ident(..)) => {}
deno>      |                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
deno>      |
deno>      = note: see issue #54883 <https://github.com/rust-lang/rust/issues/54883> for more information
deno>    Compiling swc_ecma_transforms_proposal v0.24.0
deno> error: aborting due to previous error
deno> For more information about this error, try `rustc --explain E0658`.
deno> error: could not compile `swc_ecma_transforms_optimization`
deno> To learn more, run the command again with --verbose.
deno> warning: build failed, waiting for other jobs to finish...
deno> error: build failed
error: build of '/nix/store/gnysslf3nqpcs8gcf0dwzbicclia7srl-deno-1.11.4.drv' on 'ssh-ng://builder-ng' failed: error: builder for '/nix/store/gnysslf3nqpcs8gcf0dwzbicclia7srl-deno-1.11.4.drv' failed with exit code 101
error: builder for '/nix/store/gnysslf3nqpcs8gcf0dwzbicclia7srl-deno-1.11.4.drv' failed with exit code 1
```

--- a/pkgs/development/web/deno/default.nix
+++ b/pkgs/development/web/deno/default.nix
@@ -17,15 +17,15 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "deno";
-  version = "1.11.3";
+  version = "1.11.4";
 
   src = fetchFromGitHub {
     owner = "denoland";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-Ru75eyyyaaFDD71AzDE5kgLsiB7thjqrJvvnA/PdvrA=";
+    sha256 = "sha256-0hNBIgJK4DgTK3RnFTE0E4ZM5N12HsLPo03KeSrFpeQ=";
   };
-  cargoSha256 = "sha256-8UnsmIDdFEKp4diJ6Cr/fsbPKlw7wGHmGjXu5oqh2HI=";
+  cargoSha256 = "sha256-ejEJ7Ry6HEqrfwvidC5UxyrnRx31Xd9h5VjRAoShE6Y=";
 
   # Install completions post-install
   nativeBuildInputs = [ installShellFiles ];

--- a/pkgs/development/web/deno/default.nix
+++ b/pkgs/development/web/deno/default.nix
@@ -17,15 +17,15 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "deno";
-  version = "1.11.4";
+  version = "1.11.5";
 
   src = fetchFromGitHub {
     owner = "denoland";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-0hNBIgJK4DgTK3RnFTE0E4ZM5N12HsLPo03KeSrFpeQ=";
+    sha256 = "sha256-DrmdIWaWkOQeghhsnowkdgPA+z4+L2MTs7DNyeqDi5w=";
   };
-  cargoSha256 = "sha256-ejEJ7Ry6HEqrfwvidC5UxyrnRx31Xd9h5VjRAoShE6Y=";
+  cargoSha256 = "sha256-E0qcy+/o8q9dv3KJjNkI+VNa5MBWzwla2Dz8niwvPCw=";
 
   # Install completions post-install
   nativeBuildInputs = [ installShellFiles ];

